### PR TITLE
Fix API request has 404 error when no text on beatmap ID field

### LINKED ISSUE #133: The demo beatmap card is 404 error when no text on beatmap ID field
Step to reproduce:
1. Put any beatmap ID
2. Delete to make field empty
3. 404 not found on console

CleanShot 2564-11-06 at 19 53 11@2x

This bug is not affect is the user experience but it's need to be fix.

### PR DESCRIPTION
**Pull request by HelloYeew**
Close #133 

I fix this by make the JS side request the API with beatmap ID as 0. When request as this value the API view will return the HTTPResponse as the picture like you open the add beatmap page for the first time.

### CODE REVIEW

File: beatmap_collections/static/js/live_beatmap_card.js [hunk 1]
```diff
@@ -1,6 +1,13 @@
 function RenderDemoCard(value) {
+    // If the value in form is blank, get an API call as the beatmap ID is 0
+    let api_url = '';
+    if (value === '') {
+        api_url = '/api/get_demo_card/0'
+    } else {
+        api_url = '/api/get_demo_card/' + value
+    }
```
Comment (pontakornth) [inline beatmap_collections/static/js/live_beatmap_card.js:8]: Actually, you can also write this 
```javascript
`/api/get_demo_card/${value || 0}`
```

**Review by pontakornth — APPROVED** [review]
I see no issue so I approve this.

File: beatmap_collections/static/js/live_beatmap_card.js (hunk 1 above)
Comment (HelloYeew) [reply beatmap_collections/static/js/live_beatmap_card.js:8]: I will fix to your recommendation since it's look better

**Review by SainTurDaY27 — APPROVED** [review]

**Review by Siraphop4Nene — APPROVED** [review]

**Review by some-random-idiot — APPROVED** [review]

**Comment by HelloYeew** [comment]
Let's merge it!

**Pull request merged**